### PR TITLE
Add CLI Document

### PR DESCRIPTION
**Pull request by YujiAzama**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+
+```bash
+meteoroid endpoint list
+```
+
+#### Usage example
+
+```bash
+meteoroid endpoint list
++----+----------------------------------------------------------------------------------+----------------+---------------------+-----------+--------+--------+----------+
```
Comment (ryo-ma) [inline docs/cli.md:118]: Bash syntax highlighting has been applied to the results.
I have an idea that is to split the meteoroid-cli command and the result.
What do you think?

Like this.
```bash
meteoroid endpoint list
```

```
+----+----------------------------------------------------------------------------------+----------------+---------------------+-----------+--------+--------+----------+
....
```

**Review by hide-hayashi — APPROVED** [review]
LGTM

File: docs/cli.md (hunk 1 above)
Comment (YujiAzama) [reply docs/cli.md:118]: It is certainly a bad highlight.
I'll fix.

Comment (YujiAzama) [reply docs/cli.md:118]: I removed Bash syntax highlighting.

**Pull request merged**